### PR TITLE
Adding Visual Studio and TACO-CLI to the tools list

### PR DESCRIPTION
**Pull request by cpsloal**

### CODE REVIEW

**Comment by dblotsky** [comment]
Please comment when you feel this PR is ready to be merged.

**Comment by cpsloal** [comment]
Done with this PR

**Comment by cpsloal** [comment]
Ok, done for reals now.

File: www/static/css-src/_artwork.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.iconset {
+  padding: 100px 0;
+}
+
+.button {
+    display: block;
+    background: url(/static/img/sprite.png) no-repeat;
```
Comment (dblotsky) [inline www/static/css-src/_artwork.scss]: Please prefix the URI with `{{ site.baseurl }}`, and put the URL in double-quotation marks.

File: www/static/css-src/_plugins.scss [hunk 2]
```diff
@@ -120,6 +120,13 @@
 .plugin-results-show-more {
 	text-align: center;
 	cursor:pointer;
+	padding: 10px;
+	background-color: #4CC2E4;
```
Comment (dblotsky) [inline www/static/css-src/_plugins.scss]: Please use `$brand-primary` here.

**Comment by cpsloal** [comment]
Updated code with CR feedback.

**Comment by dblotsky** [comment]
Merged. Sorry, I forgot the "This closes #339" text.

**Comment by cpsloal** [comment]
Thank you :)

**Pull request closed without merging**